src/main/kotlin/mate/academy/model/FigureList.kt

```diff
@@ -0,0 +1,9 @@
+package mate.academy.model
+
+enum class FigureList {
```

Comment:
And name of enum class should represent single value
```suggestion
enum class FigureShape {
```